package.json

```diff
@@ -30,13 +30,12 @@
   "dependencies": {
     "axios": "^1.7.4",
     "isomorphic-ws": "^4.0.1",
-    "jsonwebtoken": "^9.0.2",
+    "jose": "^5.10.0",
```

Comment:
@JJ-Cro was there a reason for using jose 5, not 6? was there an error with typescript / jest? If yes, check out the changes I made in #78 to the jest config, tsconfig and setupTEsts

Reply:
also the updated "test" scripts in the package json